geode-lucene/src/integrationTest/java/org/apache/geode/cache/lucene/LuceneIndexCreationPersistenceIntegrationTest.java

```diff
@@ -77,11 +77,11 @@ public void createCache() {
   }
 
   @Test
-  public void shouldNotUseOverflowForInternalRegionsWhenUserRegionHasOverflow() {
+  public void CanUseOverflowForInternalRegionsWhenUserRegionHasOverflow() {
```

Comment:
minor nit: method names should start with a lower-case character. Upper case should be reserved for Class/Interface names